templates/ci/python-django.yml

```diff
@@ -4,42 +4,10 @@
 
 phases:
 
-# Phase 1: Locate the Django project
-- phase: 'LocateProject'
-  queue: 'Hosted Linux Preview'
-  steps:
-
-  - task: UsePythonVersion@0
-    inputs:
-      versionSpec: '3.6'
-      architecture: 'x64'
-
-  - script: |
-      python - << 'END_DETECT'
```

Comment:
Is it common for Django projects not to be in the root?  At any rate, the project isn't moving around from build to build, and I found myself deleting this whole phase pretty early on.

Reply:
I'd say it's the default for a Django project. I couldn't find any recent discussion on the Django mailing list. [This post from 2013](https://groups.google.com/forum/#!searchin/django-users/repo$20layout%7Csort:date/django-users/J1XJD9W7yQ0/8jFFRrxVmXMJ) seems to be the last time it was really discussed there, and says you should put your project in a project directory. The Django [cookiecutter template](https://github.com/pydanny/cookiecutter-django) also does it this way.

Full Django projects are surprisingly scarce (or perhaps just hard to find) on GitHub. Mostly what's available are Django _apps_, which have a traditional `setup.py` in the root. Those are typically being turned into packages, so I'd expect people to use the other Python YAML template instead of this one.